### ISSUE TITLE
Support fort YouTube Live?

### ISSUE THREAD

**Issue by jasontucker** [rknightuk/micro-blog-lite-youtube#1]
Can you add support for YouTube live url scheme? it replaces **watch** with **live** 

An example:
https://www.youtube.com/live/ee19MfayhhM

You can see the video embedded here:
https://jasontucker.blog/2023/10/20/ep-wordstressed.html

**Comment by rknightuk** [comment]
Yep I should be able to do this by adding `live` to the regex in `lite-youtube.html`.

**Comment by jasontucker** [comment]
i'm using this over on Micro.blog for my website and many of my videos have /live/ in the url. Is this something you can add or do I need to fork and submit the plugin to Micro.blog?
